aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/automate-everything.md

```diff
@@ -250,15 +250,15 @@ In the [next chapter](source-control.md) we'll look at source code and explain w
 
 ## Resources
 
-- [Install and Configure Windows PowerShell for Azure](https://www.windowsazure.com/en-us/manage/install-and-configure-windows-powershell/). Explains how to install the Azure PowerShell cmdlets and how to install the certificate that you need on your computer in order to manage your Azure account. This is a great place to get started because it also has links to resources for learning PowerShell itself.
-- [Azure Script Center](https://www.windowsazure.com/en-us/documentation/scripts/). WindowsAzure.com portal to resources for developing scripts that manage Azure services, with links to getting started tutorials, cmdlet reference documentation and source code, and sample scripts
+- [Install and Configure Windows PowerShell for Azure](https://docs.microsoft.com/powershell/azure/overview?view=azurermps-4.3.1). Explains how to install the Azure PowerShell cmdlets and how to install the certificate that you need on your computer in order to manage your Azure account. This is a great place to get started because it also has links to resources for learning PowerShell itself.
+- [Azure Script Center](https://docs.microsoft.com/azure/automation/automation-runbook-gallery). WindowsAzure.com portal to resources for developing scripts that manage Azure services, with links to getting started tutorials, cmdlet reference documentation and source code, and sample scripts
 - [Weekend Scripter: Getting Started with Azure and PowerShell](http://blogs.technet.com/b/heyscriptingguy/archive/2013/06/22/weekend-scripter-getting-started-with-windows-azure-and-powershell.aspx). In a blog dedicated to Windows PowerShell, this post provides a great introduction to using PowerShell for Azure management functions.
-- [Install and Configure the Azure Cross-Platform Command-Line Interface](https://www.windowsazure.com/en-us/manage/install-and-configure-cli/). Getting-started tutorial for an Azure scripting framework that works on Mac and Linux as well as Windows systems.
-- [Azure Command Line Tools](https://www.windowsazure.com/en-us/downloads/#cmd-line-tools). Portal page for documentation and downloads related to command line tools for Azure.
+- [Install and Configure the Azure Cross-Platform Command-Line Interface](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest). Getting-started tutorial for an Azure scripting framework that works on Mac and Linux as well as Windows systems.
+- [Azure Command Line Tools](https://azure.microsoft.com/downloads/). Portal page for documentation and downloads related to command line tools for Azure.
```

Comment:
Should we get more specific with the link and go here instead? https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest

Reply:
Sure ... All of these link changes were performed :sparkles: *auto-magically* :sparkles: by one of my mad scripts, so I didn't manually check the vast majority of them. This is just *where* they go.

I'll change the link here, but this one could be changed in their redirect file (wherever that is :smile:). Idk who to report it to.

Reply:
Since that would just repeat the prior link, I'll fix it with a single entry.

Reply:
Hum ... not sure we should change that after all. The byline for that section is a good one for *general* (less specific) CL tools:

> Manage your Azure services and apps using scripts from the command line.